src/orb.yml

```diff
@@ -523,12 +535,14 @@ jobs:
         description: iOS executor used to run the build
         type: executor
         default: ios
+      environment_prefix:
+        description: Prefix to append to job env variables
+        type: string
+        default: ''
       pre_build:
         description: Hooks that run before building. Useful for adjusting ENV vars or running scripts.
         type: steps
-        default:
-          - set_branch_specific_env_vars
-          - create_env_file_from_vars
+        default: []
```

Comment:
just confirming [] is the default for no steps

Reply:
Yeah that's correct.